docs/README.md

```diff
@@ -94,11 +94,11 @@ If you are already familiar with SkyWalking, you could use this catalog to find
 ___
 ### Users from 5.x
 
-SkyWalking 5.x is still supported by the community. For the users planning to upgrade from 5.x to 6.x, you should know some definitions of concepts changed.
+SkyWalking 5.x is still supported by the community. For the users planning to upgrade from 5.x to 7.x, you should know some definitions of concepts changed.
 
 The most important two changed concepts are:
 
-1. Application (in 5.x) is changed to **Service** (in 6.x), also Application Instance is changed to **Service Instance**.
-2. Service (in 5.x) is changed to **Endpoint** (in 6.x).
+1. Application (in 5.x) is changed to **Service** (in 7.x), also Application Instance is changed to **Service Instance**.
+2. Service (in 5.x) is changed to **Endpoint** (in 7.x).
```

Comment:
I don't think these version numbers should be changed from `6.x` to `7.x`, because `6.x` is the nearest version (from 5.x) that has changes in concepts, the changes in this PR makes future users misunderstand that migrating from 5.x to 6.x doesn't have these issues.

@wu-sheng  If the 5.x is not supported now, I'd suggest simply remove this section, or are we still maintaining 5,.x?

Reply:
Let's remove the whole 5.x section.  

BTW, 5.x protocol should be removed too, including codes in agent, receivers, query(s). Anyone will do this? @kezhenxu94 @dmsolr @arugal

Reply:
> BTW, 5.x protocol should be removed too, including codes in agent, receivers, query(s). Anyone will do this? @kezhenxu94 @dmsolr @arugal

I'll remove them

Reply:
@kezhenxu94 I create #4176 to track this, please notice including protocol documents.

Reply:
removed